packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx

```diff
@@ -29,7 +28,7 @@ it('renders the banner that contains additional information', () => {
       dateOfBirth="May 9, 1993"
       gender="Male"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```

Comment:
We could go ahead and mock the images like the approach here: https://facebook.github.io/jest/docs/webpack.html#handling-static-assets